### PR TITLE
retrieve address from state

### PR DESCRIPTION
**Pull request by bucko13**
Small patch to get the footer address plugin working again.

### CODE REVIEW

File: webapp/plugins/local/footer-address/index.js [hunk 1]
```diff
@@ -26,7 +26,11 @@ export const decorateFooter = (Footer, { React, PropTypes }) => {
     render() {
       const { wallets, footerWidgets = [] } = this.props;
 
-      const address = wallets.primary ? wallets.primary.receiveAddress : '...';
+      const address = wallets.accountInfo
```
Comment (tynes) [inline webapp/plugins/local/footer-address/index.js]: I think that a nested ternary operator is hard to read even tho having a single expression is nice

Comment (bucko13) [reply webapp/plugins/local/footer-address/index.js]: Yeah, don't disagree. So you think something like `if (wallets.accountInfo && wallets.accountInfo.primary)` followed by assignment and an else would be better?

**Comment by bucko13** [comment]
@tynes removed nested ternary

File: webapp/plugins/local/footer-address/index.js [hunk 2]
```diff
@@ -26,15 +30,19 @@ export const decorateFooter = (Footer, { React, PropTypes }) => {
...
 
-      const address = wallets.primary ? wallets.primary.receiveAddress : '...';
-      if (address) {
-        const FooterAddress = ({ address }) => (
-          <div className="col-6 text-truncate">
-            <Text type="span">Receive Address: {address}</Text>
-          </div>
-        );
-        footerWidgets.push(widgetCreator(FooterAddress)({ address }));
+      if (wallets.accountInfo && wallets.accountInfo.primary) {
```
Comment (tynes) [inline webapp/plugins/local/footer-address/index.js]: Nice separation of logic + view

**Pull request merged**